packages/oasgraph/src/preprocessor.ts

```diff
@@ -382,17 +382,24 @@ export function createDataDef(
   if (!schema || typeof schema === 'undefined') {
     throw new Error(
       `Cannot create data definition for invalid schema ` +
-        `"${String(schema)}"`
+        `'${JSON.stringify(schema)}'`
     )
   }
 
-  const preferredName = getPreferredName(names)
-
   // resolve allOf element in schema if applicable
   if ('allOf' in schema) {
     schema = mergeAllOf(schema)
   }
 
+  const preferredName = getPreferredName(names)
+
+  const saneLinks = {}
+  if (typeof links === 'object') {
+    Object.keys(links).forEach(linkKey => {
+      saneLinks[Oas3Tools.beautify(linkKey)] = links[linkKey]
```

Comment:
We sanitize link keys now because I discovered a `productLead` and a `ProductLead` link definition in the Example API OAS. We would not report this and could have unintended/unreported consequences.

Reply:
Good find.